qe-tests/cypress/integration/models/providerocpV.ts

```diff
@@ -32,7 +33,22 @@ export class ProviderocpV extends Provider {
     clickByText(addButtonModal, addButton);
   }
 
-  delete(providerData: OcpVirtData) {
+  protected populate(providerData: OcpVirtData): void {
+    Provider.openList();
+    const { name } = providerData;
+    cy.contains(name)
+      .parent(trTag)
+      .within(() => {
+        // Validating that provider is in `Ready` state
+        cy.get(dataLabel.status, { timeout: 600 * SEC }).should('have.text', 'Ready');
+        // Verifying that network field is not empty and count is not 0
+        cy.get(dataLabel.networks).invoke('text').then(parseFloat).should('be.gt', 0);
```

Comment:
`parseFloat` is probably not necessary, you could use `parseInt`, but it doesn't really matter.